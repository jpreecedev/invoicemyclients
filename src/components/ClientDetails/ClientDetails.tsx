import React from 'react'

const ClientDetails = () => {
  return (
    <form>
      <div className="form-group narrow d-flex">
        <label htmlFor="companyName" className="w-25 d-flex align-items-center">
          <small>Company Name</small>
        </label>
        <input type="text" className="form-control" id="companyName" placeholder="Company Name" />
      </div>
      <div className="form-group narrow d-flex">
        <label htmlFor="idNumber" className="w-25 d-flex align-items-center">
          <small>ID Number</small>
        </label>
        <input type="text" className="form-control" id="idNumber" placeholder="ID Number" />
      </div>
      <div className="form-group narrow d-flex">
        <label htmlFor="website" className="w-25 d-flex align-items-center">
          <small>Website</small>
        </label>
        <input type="text" className="form-control" id="website" placeholder="Website" />
      </div>
      <div className="form-group narrow d-flex">
        <label htmlFor="vatNumber" className="w-25 d-flex align-items-center">
          <small>VAT Number</small>
        </label>
        <input type="text" className="form-control" id="vatNumber" placeholder="VAT Number" />
      </div>
      <div className="form-group narrow d-flex">
        <label htmlFor="phoneNumber" className="w-25 d-flex align-items-center">
          <small>Phone Number</small>
        </label>
        <input type="text" className="form-control" id="phoneNumber" placeholder="Phone Number" />
      </div>
    </form>
  )
}

export { ClientDetails }
