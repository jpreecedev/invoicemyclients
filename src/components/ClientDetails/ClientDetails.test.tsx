import React from 'react'
import { render } from '@testing-library/react'
import { ClientDetails } from './ClientDetails'

describe('<ClientDetails /> tests', () => {
  it('should match snapshot', () => {
    expect(render(<ClientDetails />).container).toMatchSnapshot()
  })
})
