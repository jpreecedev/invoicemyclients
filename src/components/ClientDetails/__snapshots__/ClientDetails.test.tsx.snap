// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ClientDetails /> tests should match snapshot 1`] = `
<div>
  <form>
    <div
      class="form-group narrow d-flex"
    >
      <label
        class="w-25 d-flex align-items-center"
        for="companyName"
      >
        <small>
          Company Name
        </small>
      </label>
      <input
        class="form-control"
        id="companyName"
        placeholder="Company Name"
        type="text"
      />
    </div>
    <div
      class="form-group narrow d-flex"
    >
      <label
        class="w-25 d-flex align-items-center"
        for="idNumber"
      >
        <small>
          ID Number
        </small>
      </label>
      <input
        class="form-control"
        id="idNumber"
        placeholder="ID Number"
        type="text"
      />
    </div>
    <div
      class="form-group narrow d-flex"
    >
      <label
        class="w-25 d-flex align-items-center"
        for="website"
      >
        <small>
          Website
        </small>
      </label>
      <input
        class="form-control"
        id="website"
        placeholder="Website"
        type="text"
      />
    </div>
    <div
      class="form-group narrow d-flex"
    >
      <label
        class="w-25 d-flex align-items-center"
        for="vatNumber"
      >
        <small>
          VAT Number
        </small>
      </label>
      <input
        class="form-control"
        id="vatNumber"
        placeholder="VAT Number"
        type="text"
      />
    </div>
    <div
      class="form-group narrow d-flex"
    >
      <label
        class="w-25 d-flex align-items-center"
        for="phoneNumber"
      >
        <small>
          Phone Number
        </small>
      </label>
      <input
        class="form-control"
        id="phoneNumber"
        placeholder="Phone Number"
        type="text"
      />
    </div>
  </form>
</div>
`;
